app/Models/Forum/Topic.php

```diff
@@ -80,8 +80,8 @@ public static function createNew($forum, $params, $pollParams = null)
             }
 
             if (($params['cover'] ?? null) !== null) {
```

Comment:
yes. That's why for the longest time, `isset(literal)` was a parse error. It now has a better error message now, though.